online-store/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "online-store",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.base.config.js",
+  "scripts": {
```

Comment:
JFYI `"type-check": "tsc --noEmit"` 
I am usually adding this script to check if I have ts errors in my whole project